server/db/migrations/20230309195953_update-campaign-model.js

```diff
@@ -0,0 +1,17 @@
+const tableName = 'campaigns'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.alterTable(tableName, (table) => {
+      // Simple fields
+      table.string('campaign_tagline').nullable()
+      table.string('campaign_text').nullable()
+      table.string('supplemental_text').nullable()
+    })
+  },
+
+  async down(knex) {
+    // Drop the table
+    await knex.schema.dropTable(tableName)
```

Comment:
I don't think it should drop the table here, Sarah.

Tip: `down(knex)` is supposed to undo the updates made by `up(knex)`. 😉